### DISCUSSION TITLE
AWS RDS compatability

### DISCUSSION THREAD

**Discussion by mal-drop** [timgit/pg-boss#349] [Q&A] [answered]
Just want to check about pg-boss compatibility with AWS Postres RDS and Aurora?  Anyone using pg-boss with those databases, anything to be aware of.
Thank you for your help.
regards Malcolm

**Comment by timgit** [comment, marked as answer]
RDS yes, since it uses the postgres storage engine.  Aurora I haven't tested but I doubt it works since it uses a proprietary storage engine.
